assets/locales/pages/settings/notification.json

```
@@ -3,12 +3,14 @@
     "title": "Notification",
     "receiveWeekReport": "Receive weekly report",
     "receiveMonthReport": "Receive monthly report",
+    "receiveReminder": "Remind activity you forgot to stop",
     "updated": "Notification updated"
   },
   "ja": {
     "title": "メール配信を設定",
     "receiveWeekReport": "週次レポートを受信する",
     "receiveMonthReport": "月次レポートを受信する",
+    "receiveReminder": "停止し忘れた計測をリマインドする",
```

個人的にですが、`止め忘れた計測` の方がしっくりくるかもと思いました 😄 

```suggestion
    "receiveReminder": "止め忘れた計測をリマインドする",
```